### PR TITLE
Update 1-month-down.md

### PR DESCRIPTION
**Pull request by boshnivolo**

### CODE REVIEW

**Comment by jmcreasman** [comment]
Enjoyed reading these, I can relate to trying to turn the logic in my mind into actual code

**Comment by rosalsm** [comment]
Easy to read and fun :+1:

File: how-to-for-you.md [hunk 1]
```diff
@@ -1,18 +1,27 @@
 ### THIS WEEK: HOW TO...WRITE A REALLY GOOD TUTORIAL!
```
Comment (kellymurray) [inline how-to-for-you.md:2]: Focus on a technical tutorial for next week, please.

File: how-to-for-you.md [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+This helps your readers make the distinction between your explaination and the code they may need to input.
 
-<h4>STEP 4:</h4> PROFIT (NOT REALLY BECAUSE WE'RE PAYING TO BE HERE, SO LIKE, NEGATIVE PROFIT, BUT STILL...)
+<h4>STEP 5:</h4> PROOFREAD! 
 
-#### Ideas for tutorial this week: 
-* node-sass and you
-* sass
-* linking properly
+(Also, small note to Kelly, who has to read this: I don't think I should teach anybody anything because every time I do you end up with thousands of commits from node_modules and sudo disasters. Just saying....)
```
Comment (kellymurray) [inline how-to-for-you.md:25]: Use `.gitignore`! Don't `git add .` or `git add -a`! :)

**Comment by ericquayson** [comment]
:+1:  Nice journal and tutorial. "Week 5's yak-shaving assignments feel like precious moments of relief on the stretcher as the paramedics wisk me away." -- this statement is very true.

**Comment by jmcreasman** [comment]
Loved your comment on proofreading :+1:

**Comment by Ricecoder** [comment]
Your tutorial was a good primer on making our tutorials better. Good job! Maybe next week you could do a tutorial on psuedo elements... :+1:

**Comment by rosalsm** [comment]
I love reading your journal entries and tutorials. This time you give us a good base to start a good tutorial :+1:

**Pull request merged**